packages/yoga/src/Divider/web/Divider.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import theme from '../../Theme/helpers/themeReader';
+
+const StyledDivider = styled.div`
+  ${props => {
+    const { borders, colors } = theme(props);
+
+    return css`
+      flex: 1;
+      height: ${borders.small}px;
+      background-color: ${colors.light};
```

Comment:
Could we use this same approach at the `native`? Avoiding have a container there

Reply:
I tried to do the same approach on `native`, but unfortunately, it not had work. Have some idea?